examples/with-nextjs-and-clerk-auth/src/components/ThemeRegistry/RootThemeProvider.tsx

```diff
@@ -1,25 +1,131 @@
 'use client';
 
 import * as React from 'react';
+import { PropsWithChildren, useCallback, useMemo } from 'react';
```

Comment:
React imports are duplicated, and we probably don't need the whole `React` object